src/h5web/App.tsx

```diff
@@ -45,11 +45,7 @@ function App(): JSX.Element {
           style={{ display: isExplorerOpen ? undefined : 'none' }}
         />
 
-        <ReflexElement
-          className={styles.mainArea}
-          flex={isExplorerOpen ? 0.75 : 1}
-          minSize={500}
-        >
+        <ReflexElement className={styles.mainArea} flex={75} minSize={500}>
```

Comment:
We had a ternary before `flex={isExplorerOpen ? 0.75 : 1}` that is now missing. Not sure if that's on purpose.

Reply:
Yeah it works without it now. Since it's the only flex item when the explorer is hidden, it grows to fit the space. It wasn't working before for some reason I never understood.